public/documentation/windows/run_on_windows.md

```diff
@@ -1,28 +1,13 @@
 # Run a Crosswalk App on Windows
 
-## Run using Crosswalk packaging tool
+## Run a packaged Crosswalk application
 
-1. Download latest Crosswalk binary from: https://download.01.org/crosswalk/releases/crosswalk/windows/
-
-   (Note that this step will not be required when Crosswalk For Windows binaries are published on download.01.org)
-
-2. In your command line, inside the directory of the application you want to build run:
-
-   ```
-    > crosswalk-pkg \
-        --platforms=windows \
-        --crosswalk=C:\Users\alexisme\Downloads\crosswalk-15.44.374.0.zip \
-        xwalk-simple
-   ```
-
-   This will package the application defined in the specified manifest.json file and produce an .msi (in our example `com.app.simple-0.1.0.0.msi`). The .msi is currently a 32-bit build of Crosswalk for Windows and should run fine on both 32-bit and 64-bit Windows.
-
-3. Click on the .msi and follow the installation instructions.
-4. After the installation successfully finishes you should be able to see your application in the start menu. Click on it and launch the application.
+1. Click on the .msi produced following the [build an application](build_an_application) section and follow the installation instructions.
+2. After the installation successfully finishes you should be able to see your application in the start menu. Click on it and launch the application.
```

Comment:
Maybe "click on it to launch your program" to avoid having "application" twice in the same sentence?